SAMPLing_instructions.md

```diff
@@ -95,5 +95,6 @@ All the host-guest system files in the `SAMPLing/` directory were prepared using
 - Hosts and guests were both parametrized with GAFF v1.8 and antechamber. AM1-BCC charges were generated using OpenEye's QUACPAC toolkit through `openmoltools 0.8.1`.
 - The systems were solvated in a 12A buffer of TIP3P water molecules using tleap. ParmEd `2.7.3` was used to remove some of the water molecules from the OA complexes to reduce them to have the same number of waters.
 - The systems' net charge was neutralized with Na+ and Cl- ions. More Na+ and Cl- ions were added to reach the ionic strength of 60mM for OA/TEMOA systems and 150mM for CB8. Note that this ionic strength is different from the one used for the experimental measurements.
-- The system was minimized with the L-BFGS optimization algorithm and equilibrated by running 1ns of Langevin dynamics (BAOAB splitting, 1fs time step) at 298.15K with a Monte Carlo barostat set at 1atm using `OpenMM 7.1.1`. PME was used for long-range electrostatic interactions with a cutoff of 10A. VdW interactions used the same 10A cutoff and a switching distance of 9A.
-- After the equilibration, the `System` was serialized into the OpenMM `xml` format. The `rst7` file was generated during the equilibration using the `RestartReporter` in the `parmed.openmm` module. The AMBER `prmtop` and `rst7` files were then converted to GROMACS `top`/`gro` and PDB formats by ParmEd and MDTraj `1.9.1` respectively.
+- The system was minimized with the L-BFGS optimization algorithm and equilibrated by running 1 ns of Langevin dynamics (BAOAB splitting, 1 fs time step) at 298.15K with a Monte Carlo barostat set at 1 atm using `OpenMM 7.1.1`. PME was used for long-range electrostatic interactions with a cutoff of 10A. VdW interactions used the same 10A cutoff and a switching distance of 9A.
+- After the equilibration, the `System` was serialized into the OpenMM `xml` format. The `rst7` file was generated during the equilibration using the `RestartReporter` in the `parmed.openmm` module (ParmEd `2.7.3`). The AMBER `prmtop` and `rst7` files were then converted to PDB format by MDTraj `1.9.1`.
+- Other coversions are described in `SAMPLing/conversion.md`, as well as validations of the energies calculated at each configuration between programs.
```

Comment:
"coversions" -> "conversions"

Reply:
Also, "validations of the energies calculated at each configuration between programs" is confusing; do you mean "after each CONVERSION between programs"? If not, what are the configurations in question?